### PR TITLE
OCPBUGS-29656:scaling-worker-latency-profiles document issues

### PR DESCRIPTION
**Pull request by GroceryBoyJr**
Version(s): 4.15+

Issue: https://issues.redhat.com/browse/OCPBUGS-29656?filter=-1

Link to docs previews:
https://84762--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-worker-latency-profiles.html
https://84762--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/cluster-tasks.html
https://84762--ocpdocs-pr.netlify.app/openshift-enterprise/latest/scalability_and_performance/scaling-worker-latency-profiles.html

QE review:
- [X] QE has approved this change. (QE REQUESTED these changes)

Additional information: Qiujie Li , Quality Engineer, found some small but important errors in previously produced documentation. The requested corrections are shown in the issue linked above.  I have also corrected pre-existing issues in the source files that were flagged by vale. 

REVIEWERS PLEASE NOTE: vale returns errors on the capitalization of the word "Kubelet". It wants the word to begin lowercase, but in this situation, it's part of the "Kubelet Controller Manager", so it must be capitalized.

### CODE REVIEW

File: modules/nodes-cluster-worker-latency-profiles-using-at-creation.adoc [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 [IMPORTANT]
 ====
-To edit the configuration of the installer, you will first need to use the command `openshift-install create manifests` to create the default node manifest as well as other manifest YAML files. This file structure must exist before we can add workerLatencyProfile. The platform on which you are installing may have varying requirements. Refer to the Installing section of the documentation for your specific platform.
+To edit the configuration of the installer, first use the command `openshift-install create manifests` to create the default node manifest and other manifest YAML files. This file structure must exist before you can add `workerLatencyProfile`. The platform on which you are installing might have varying requirements. Refer to the Installing section of the documentation for your specific platform.
```
[the remark this answers is not in the document]
Comment (GroceryBoyJr) [reply modules/nodes-cluster-worker-latency-profiles-using-at-creation.adoc:11]: Not sure this vale-identified issue is correct for my context.

[the remark this answers is not in the document]
Comment (skopacz1) [reply modules/nodes-cluster-worker-latency-profiles-using-at-creation.adoc:11]: This seems like it refers to the process of installing a cluster, in which case it does seem like "installation program" is the appropriate word choice imo. In the installer docs we use that term

File: snippets/worker-latency-profile-intro.adoc [hunk 2]
```diff
@@ -25,6 +25,6 @@ This behavior can cause problems if your network is prone to latency issues, esp
 
 To avoid this problem, you can use _worker latency profiles_ to adjust the frequency that the `Kubelet` and the Kubernetes Controller Manager wait for status updates before taking action. These adjustments help to ensure that your cluster runs properly if network latency between the control plane and the worker nodes is not optimal.
 
-These worker latency profiles contain three sets of parameters that are pre-defined with carefully tuned values to control the reaction of the cluster to increased latency. No need to experimentally find the best values manually.
+These worker latency profiles contain three sets of parameters that are predefined with carefully tuned values to control the reaction of the cluster to increased latency. No need to experimentally find the best values manually.
```
Comment (skopacz1) [inline snippets/worker-latency-profile-intro.adoc:28]: If you're touching up the docs, I might suggest rewording the last sentence to sound a tad more formal:
```suggestion
These worker latency profiles contain three sets of parameters that are predefined with carefully tuned values to control the reaction of the cluster to increased latency. There is no need to experimentally find the best values manually.
```

**Review by skopacz1 — COMMENTED** [review]
Just a few thoughts, otherwise LGTM

File: snippets/worker-latency-profile-intro.adoc (hunk 2 above)
Comment (GroceryBoyJr) [reply snippets/worker-latency-profile-intro.adoc:28]: Great suggestion, @skopacz1 , glad to do that!
